src/rexsdesktop/controller/User_.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package rexsdesktop.controller;
+
+import java.awt.image.BufferedImage;
+import rexsdesktop.CurrentUser;
+import rexsdesktop.model.UserModel;
+
+/**
+ *
+ * @author Eduardo
+ */
+public class User_{
+    
+    private int idUsuario;
+    private String nombreCompleto;
+    private String email;
+    private String hash;
+    private BufferedImage fotoPerfil;
+    private int idTipoUsuario;
+    private int idEstadoUsuario;
+    /*public User_(int id, String nombre, String email, String hash, BufferedImage foto, int idTipoUsuario, int idEstadoUsuario){

```

Comment:
remove this unused code. Always try to remove your unused code

Reply:
Okay, this is done.